packages/noya-designsystem/src/utils/getGradientBackground.tsx

```diff
@@ -0,0 +1,19 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { sketchColorToRgbaString } from '..';
+
+export function getGradientBackground(
+  value: Sketch.GradientStop[],
+  type?: Sketch.GradientType,
+) {
+  const color = [...value]
+    .sort((a, b) => a.position - b.position)
+    .map((g) => `${sketchColorToRgbaString(g.color)} ${g.position * 100}%`)
+    .join(', \n');
+
+  const position =
```

Comment:
Can we use a `switch` here?

```ts
switch (type) {
  case Sketch.GradientType.Radial: 
    return `radial-gradient(circle, ${color})`
}
```

Reply:
Can we add angular/conic gradient as an option? https://developer.mozilla.org/en-US/docs/Web/CSS/conic-gradient()

If it's hard, we can come back to it